csss-site/src/events/frosh/templates/frosh/frosh_week.html

```diff
@@ -5,17 +5,24 @@
 {% load staticfiles %}
 
 {% block content %}
-    <h2 class="title is-size-2">Frosh Week</h2>
-    <p class="lead">FroshWeek is the ultimate introductory experience for Computing Science students at Simon Fraser University.</p>
-    <p class="lead">Organized by us, the CSSS, in partnership with the School of Computing Science, it is a week of fun-filled activities designed to orient, entertain and educate new university students with a Computing Science twist.</p>
-    <p class="lead">Held the week right before the start of the Fall Semester {% comment %}<b>(August 31st - Sept 4th)</b>{% endcomment %}, FroshWeek is organized by students, for students. Incoming students receive: an in-depth introduction to the facilities available to them as Computing Science students; many entertaining on- and off-campus challenges and diversions to motivate, amuse, and connect them with other students; valuable advice from their elders with respect to courses and the university experience in general; and some tools to make the transition to university much smoother.</p>
-	<p class="lead">Check out <a href="{%  static 'frosh_static/Under-Construction1.jpg' %}">The Official Frosh Week Site</a> for more info, and to register!</p>
-{#    <p class="lead">Check out <a href="{{ URL_ROOT }}events/frosh/2020">The Official Frosh Week Site</a> for more info, and to register!</p>#}
-    <h3 class="title is-size-2">The RAMing Ceremony</h3><hr>
-    <p class="lead">Our most hallowed of rituals. It doesn't hurt, we promise.</p>
-    <h3 class="title is-size-2">Pac Macro</h3>
-    <hr><p class="lead">The Frosh Week classic: become Pacman himself among the streets of downtown Vancouver. Pacman and the ghosts chase each other amidst the crowds and traffic, all the while being directed by their team mates, stationed at Harbour Centre. There, they have a live overview of the positions of all players, projected on a large screen, and managed by custom CSSS software.</p>
-    <h3 class="title is-size-2">Midnight Madness</h3>
-    <hr>
-    <p class="lead">An all-night event in the ASB, where the four Applied Science schools (Comp Sci, Engineering, Software Systems, and Mechtronics) go head-to-head in various challenges, until they arrive at the conclusion that everyone knew already: Comp Sci is the best.</p>
+    <section class="section">
```

Comment:
The current https://sfucsss.org/events/frosh/ doesn't have any margins, so the text content is being displayed from side to side.  The only actual change here is the addition of lines 8 and line 27 which surrounds the content in a section with proper margins. Every line in between is the exact same. Not sure why Git counts it as different code. 

tl;dr theres only one actual change here, which adds proper margins to the page. Every other detected change to the file is a false positive.